site/Lightbox.tsx

```diff
@@ -8,32 +8,60 @@ import {
     faPlus,
 } from "@fortawesome/free-solid-svg-icons"
 import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch"
-import { LoadingIndicator } from "@ourworldindata/components"
+import { LoadingIndicator, BodyDiv } from "@ourworldindata/components"
 import { triggerDownloadFromBlob } from "@ourworldindata/utils"
-import { unmountComponentAtNode } from "react-dom"
 import { useTriggerOnEscape } from "./hooks.js"
+import cx from "classnames"
+
+const LightboxImage = ({
+    src,
+    alt,
+    isLoaded,
+    setIsLoaded,
+    width,
+    height,
+}: {
+    src: string
+    alt: string
+    isLoaded: boolean
+    setIsLoaded: any
+    width: number
+    height: number
+}) => {
+    return (
+        <img
+            width={width}
+            height={height}
+            onLoad={() => setIsLoaded(true)}
+            className={cx({
+                "lightbox__img--is-svg": src.endsWith(".svg"),
```

Comment:
I just checked and we don't have any published SVG's - we could remove this if we wanted.